### PR TITLE
Fix travis.yml

### PR DESCRIPTION
**Pull request by briandds**
This fixes the yaml in `.travis.yml`

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,5 +1,8 @@
 language: python
-  python:
-    - "3.6"
-  script:
-    - python -m pytest
+python:
+  - "3.6"
+before_install:
+  - gem install sass
```
Comment (jordangov) [inline .travis.yml]: Where is the `sass` gem invoked? That is, where do we call for the sass to be compiled into css?

Comment (briandds) [reply .travis.yml]: The sass CLI is used by webassets -- when the url is requested, if the css doesn't exist then it is generated.

Comment (briandds) [reply .travis.yml]: i.e. when the `.urls` method is called (not when the HTTP request comes in)

Comment (briandds) [reply .travis.yml]: https://github.com/dod-ccpo/atst/blob/master/templates/base.html.to#L7

Comment (jordangov) [reply .travis.yml]: Interesting... ok, wasn't aware of that!

**Pull request merged**